grids/AccountsGridView.php

```diff
@@ -254,13 +254,35 @@ public function init()
                             ]);
                         }
                     }
-
+                    if ($model->space != null && AccountHelper::canManageAccount($model) && $model->account_type == Account::TYPE_TASK) {
```

Comment:
what is the differences between the if bloc and the else bloc other than the message ?